packages/graphql-transformer-core/src/validation.ts

```diff
@@ -64,13 +63,13 @@ export const specifiedRules = [
     ScalarLeafs,
     FieldsOnCorrectType,
     KnownDirectives,
-    UniqueDirectivesPerLocation,
```

Comment:
We are disabling this validation as we are now allowing multiple directives of the same type to be placed on the same field. This is the direction moving forward in future versions of GraphQL.

Reply:
Can we decorate the directives themselves which can say if they can be used only once per field or if they can be used multiple times?

Reply:
Unfortunately not yet but that may be coming to future graphql versions.